test/lolcommits_test.rb

```diff
@@ -12,50 +12,7 @@
 
 include Lolcommits
 
-class LolTest < Test::Unit::TestCase
-  def test_can_parse_git
```

Comment:
:+1: its probably Ok to drop this test, I'd like to think most of the features would fail if GitInfo couldn't be read...